app/viewmodels/AddressFormattingHelper.scala

```diff
@@ -18,8 +18,20 @@ package viewmodels
 
 import models.backend.UkAddress
 import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.govukfrontend.views.Aliases.HtmlContent
 
 object AddressFormattingHelper {
+
+  def formatBusinessAddress(ukAddress: UkAddress, tradingName: Option[String]): HtmlContent = {
+    HtmlContent(tradingName.fold("")(tradingName => tradingName + "<br/>") +
```

Comment:
Will think if there is a slightly cleaner way to do this

Reply:
Filter ukAddresses by empty lines and then join using "<br/>"? The bit in the middle might be a little cleaner then